files/galaxy/tpv/tools.yml

```diff
@@ -1060,21 +1060,21 @@ tools:
       require:
         - singularity
 
-  toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/3.0.6.*:
-    mem: 4
+  toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/.*:
     cores: 6
-    env:
-      GENEMARK_PATH: "/usr/local/tools/_conda/envs/__braker3@3.0.6/bin/genemark/bin/"
-      PROTHINT_PATH: "/usr/local/tools/_conda/envs/__braker3@3.0.6/bin/genemark/bin/gmes/ProtHint/bin/"
-
-  toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/3.0.3.*:
     mem: 4
-    cores: 6
     params:
-      singularity_container_id_override: docker://quay.io/bgruening/braker3:v.1.0.4
-    scheduling:
-      require:
-        - singularity
+      singularity_enabled: true
```

Comment:
I think the +galaxy suffixes are needed for the tool versions here, right?
Looks like braker3 is not working at EMBO training

Reply:
https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1094